rosetta/cosmos/launchpad/network.go

```diff
@@ -0,0 +1,26 @@
+package launchpad
+
+import (
+	"context"
+
+	"github.com/coinbase/rosetta-sdk-go/types"
+)
+
+func (l Launchpad) NetworkList(ctx context.Context, request *types.MetadataRequest) (*types.NetworkListResponse, *types.Error) {
```

Comment:
```suggestion
func (l *Launchpad) NetworkList(ctx context.Context, request *types.MetadataRequest) (*types.NetworkListResponse, *types.Error) {
```

and same change for similar places?

Reply:
I don't have specific preference, normally is safer to go without pointers, and in this case we dont change the state of the adapter so I would go this way until needed.

In terms of performance I have seen counterituitive reports that say that is faster to pass by value in many cases, so dont have specific preference.